src/connectedk8s/azext_connectedk8s/clientproxyhelper/_utils.py

```diff
@@ -146,8 +146,7 @@ def fetch_and_post_at_to_csp(
     profile = Profile(cli_ctx=cmd.cli_ctx)
     try:
         credential, _, _ = profile.get_login_credentials(
-            subscription_id=profile.get_subscription()["id"],
-            resource=consts.KAP_1P_Server_App_Scope,
```

Comment:
Track 1 Azure Python SDK support has been dropped in Azure CLI 2.70.0 (https://github.com/Azure/azure-cli/issues/30951).

https://github.com/Azure/azure-cli/issues/30951 explains why `resource` parameter should not be passed to `Profile.get_login_credentials()`.